view/theme/frio/php/default.php

```diff
@@ -28,6 +28,27 @@
 			if(x($page,'htmlhead')) echo $page['htmlhead'];
 		}
 	?>
+	<?php
+		require_once("view/theme/frio/php/schema.php");
```

Comment:
Do we need `schema.php` here? `schema.php` does provide the `function get_schema_info()` but we don't use the function in this file.